Changed is_residential field for SmallHotel to yes

anchapin opened the pull request:
This change was made so that the correct envelope parameters would be used for the Baseline model.

jmarrec commented on lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/data/179d_ashrae_90_1_2007.schedules.json:
```diff
@@ -2624,4 +2624,4 @@
     }
   ]
-}
+}
```
Our linter for end of file (pre-commit) on bem to surrogate gem will complain about this change.

anchapin committed:
Changed is_residential field for SmallHotel to yes

This change was made so that the correct envelope parameters would be used for the Baseline model.

jmarrec commented:
I reverted the end of file changes, keeping only the is_residential one. Modified slightly the commit message, and attributed it back to you.

A unit test could have been a good idea FYI.
